conf/common.cfg

```diff
@@ -0,0 +1,27 @@
+[db]
```

Comment:
I'd rather not create directories that aren't standard Django, so can this just go in `monitoring`?

Also, I'd rather the name was clearer to its purpose so perhaps `settings.ini` (as it's actually INI format), and a comment at the top explaining that it's parsed by `settings.py`.

Reply:
Sure Adrian. Added into the monitoring folder itself with `settings.ini`.